browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchPersistentStore.java

```diff
@@ -0,0 +1,42 @@
+package com.braintreepayments.browserswitch;
+
+import android.content.Context;
+
+import org.json.JSONException;
+
+public class BrowserSwitchPersistentStore {
```

Comment:
Yeah technically there should only be one request at a time. The BrowserSwitchClient enforces this externally by calling `getActiveRequest` and `putActiveRequest`. The idea was to match the previous implementation and help ease the strain of migration.

Reply:
Here: https://github.com/braintree/browser-switch-android/blob/63e709ac090ff39097798ed3bccf9da345a180c3/browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchClient.java#L104

Reply:
Also yeah this class was added because it made testing really easy since its only concern is to persist BrowserSwitch requests so they can be restored when the app returns to the foreground.